src/main/java/com/videoclub/suriken/service/impl/MovieServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package com.videoclub.suriken.service.impl;
+
+import com.videoclub.suriken.model.Movie;
+import com.videoclub.suriken.model.MovieRenter;
+import com.videoclub.suriken.repository.MovieRepository;
+import com.videoclub.suriken.repository.RenterRepository;
+import com.videoclub.suriken.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+
+    private final MovieRepository movieRepository;
+    private final RenterRepository renterRepository;
+
+    @Autowired
+    public MovieServiceImpl(MovieRepository movieRepository, RenterRepository renterRepository) {
+        this.movieRepository = movieRepository;
+        this.renterRepository = renterRepository;
+    }
+
+
+    @Override
+    public void addMovieToDb(Movie movie) {
+        movie.setMovGenre(Movie.Genre.Drama);
+        movieRepository.save(movie);
+    }
+
+    @Override
+    public List<Movie> getAllMovies() {
+        List<Movie> movies = movieRepository.findAll();
+
+        movies.forEach(movie -> System.out.println("Movie" + movie.getMovName() + "has :" + movie.getMovieRenters().size()));
```

Comment:
Cisto da napomenem. Nikada se u pravom kodu ne koristi system.out.println, uvek se koriste logovi, kao slf4j ili log4j. Da se koristi lombok, imao bi to 'out of the box'. Baci gugl kako da se loguje u spring boot-u, treba da se sklone println-ovi

Reply:
Hocu hocu obavezno. I u ovom mom poslu ima taj standardni output koji bukvalno ne sme da se koristi nego se koristi log. Guglam to odmah i sklanjam!